packages/runtime/container-runtime/package.json

```diff
@@ -230,6 +230,31 @@
 		"typescript": "~5.4.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"Class_ChannelCollection": {
```

Comment:
@tylerbutler - You mentioned that soon we won't be doing type tests for `@internal` things (like this example) - any ETA on that?